tests/wpt/mozilla/tests/mozilla/interfaces.js

```diff
@@ -80,7 +80,7 @@ function test_interfaces(interfaceNamesInGlobalScope) {
       assert_true(name in interfaceMap,
                   "If this is failing: DANGER, are you sure you want to expose the new " +
                   "interface " + name + " to all webpages as a property on the global? " +
-                  "Do not make a change to this file without review from jdm or Ms2ger " +
+                  "Do not make a change to this file without review" +
```

Comment:
All changes need a review, so this makes the statement a bit redundant. Maybe we should agree on a policy for when to expose new interfaces and write that here?

Reply:
I think we can do that separately and mention it in the Servo book or somewhere else.

For the assert message, maybe I can change it to just:

```
"If this is failing: DANGER, are you sure you want to expose the new " +
interface " + name + " to all webpages as a property on the global?"
```

What do you think? I'm not sure if we want to provide a policy in the assert message anyway.

Reply:
We should use CODEOWNERS to ensure knowledgeable people review such PR.

Reply:
> I think we can do that separately and mention it in the Servo book or somewhere.
> 
> For the assert message, maybe I can change it to just:
> 
> ```
> "If this is failing: DANGER, are you sure you want to expose the new " +
> interface " + name + " to all webpages as a property on the global?"
> ```
> 
> What do you think? I'm not sure if we want to provide a policy in the assert message anyway.

I like this change.

Reply:
> What do you think? I'm not sure if we want to provide a policy in the assert message anyway.

Seems fine to me if @jdm is okay as well.